net/dns-over-https/Makefile

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2021 Martin Schneider <martschneider@google.com>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dns-over-https
+PKG_SOURCE_DATE:=2021-09-29
+PKG_SOURCE_VERSION:=b338c7ae52bed0d1a5e85e30c76d023efb46580d
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/m13253/dns-over-https
+PKG_MIRROR_HASH:=ab15dbefac0c9bb0d5763749b532284568002d5b4796ff61b8ca3b51cae771e7
+
+PKG_MAINTAINER:=Martin Schneider <martschneider@google.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
```

Comment:
For go packages we typically also include `include ../../lang/golang/golang-package.mk`.